### PR TITLE
Fix DOCS

### PR DESCRIPTION
**Pull request by TharonWeighill**
Should contain all fixes to installation guides.

### CODE REVIEW

File: docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc [hunk 1]
```diff
@@ -190,10 +190,10 @@ Before starting CleanSpeak Management Interface for the first time, you'll need
 The configuration file may be found in the following directory, assuming you installed it in the default locations. If you have installed it in an alternate location, the path to this file will be different.
 
 Windows::
-  `C:\inversoft\config`
+  `C:\cleanspeak\config`
 
 Mac OS X or Linux::
-  `/usr/local/inversoft/config`
+  `usr/local/cleanspeak/config`
```
Comment (robotdan) [inline docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc:196]: Missing a leading slash

File: docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc [hunk 2]
```diff
@@ -190,10 +190,10 @@ Before starting CleanSpeak Management Interface for the first time, you'll need
 The configuration file may be found in the following directory, assuming you installed it in the default locations. If you have installed it in an alternate location, the path to this file will be different.
 
 Windows::
-  `C:\inversoft\config`
+  `C:\cleanspeak\config`
 
 Mac OS X or Linux::
```
Comment (robotdan) [inline docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc:195]: `Mac OS X` -> `macOS`

**Comment by robotdan** [comment]
You don't want to commit the `.shelf` changes, you can remove those from the PR.

File: docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc [hunk 3]
```diff
@@ -215,4 +215,4 @@ database.username=cleanspeak
 database.password=cleanspeak
 ----
 
-CleanSpeak WebService should now be configured, the database should be created, and everything should be ready to run. You can start CleanSpeak WebService using the instructions in the [start-cleanspeak-webservice] section above.
\ No newline at end of file
+CleanSpeak WebService should now be configured, the database should be created, and everything should be ready to run. You can start CleanSpeak WebService using the instructions in the Start CleanSpeak Webservice section above.
```
Comment (robotdan) [inline docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc:218]: This is still broken.

File: docs/3.x/tech/installation-guide/system-requirements.adoc [hunk 4]
```diff
@@ -29,7 +29,7 @@ If you do not intend to store content, which means you’ll either be using the
 CleanSpeak is capable of running on most modern operating systems. Below is a list of the supported operating systems.
 
 * Linux - all distributions (64-bit)
-* Mac OS X 10.8 (Mountain Lion) or newer
+* macOS  10.8 (Mountain Lion) or newer
 * Windows Server 2008 SP2 (64-bit) w/ Windows Management Framework 3.0 or newer
 * Windows Server 2008 R2 (64-bit) w/ Windows Management Framework 3.0 or newer
 * Windows 7 SP1 (64-bit) w/ Windows Management Framework 3.0 or newer
```
Comment (robotdan) [inline docs/3.x/tech/installation-guide/system-requirements.adoc:35]: This needs to be updated. Windows 7 is not longer in support by Microsoft, review the rest of this as well.

File: docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc [hunk 5]
```diff
@@ -150,12 +150,12 @@ These instructions will assist you in editing the CleanSpeak WebService configur
...
 ====
 
-In the following examples, <root_user> is the name of the root user for your database. The <root_user>` must be either the root user or a user that has privileges to create databases. For MySQL, this is generally a user named 'root'. On PostgreSQL, this is generally a user named 'Postgres'. Run the following SQL commands to configure the database for use by CleanSpeak.
+In the following examples, `<root_user>` is the name of the root user for your database. The `<root_user>` must be either the root user or a user that has privileges to create databases. For MySQL, this is generally a user named `root`. On PostgreSQL, this is generally a user named 'postgres'. Run the following SQL commands to configure the database for use by CleanSpeak.
 
 [source,shell]
 .MySQL
 ----
-# Create the cleanspeak database, replace <root_user> a valid superuser.
+# Create the cleanspeak database, replace `<root_user>` a valid superuser.
```
Comment (robotdan) [inline docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc:158]: You don't need back ticks here, this is already in code.

File: docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc (hunk 3 above)
Comment (robotdan) [inline docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc:218]: Fix this link

File: docs/3.x/tech/apis/_user-request-body.adoc [hunk 6]
```diff
@@ -18,7 +18,7 @@ endif::[]
 |The email address of the user.
 
 |[field]#user.expirationInstantUTC# [type]#[Long]# [optional]#Optional#
-|The link:../reference/data-types#instants[instants] this user will expire and will be unable to login to CleanSpeak.
+|The `link:../reference/data-types#instants[instants]` this user will expire and will be unable to login to CleanSpeak.
```
Comment (robotdan) [inline docs/3.x/tech/apis/_user-request-body.adoc:21]: Remove back ticks

File: docs/3.x/tech/apis/admins-moderators.adoc [hunk 7]
```diff
@@ -93,7 +93,7 @@ include::docs/3.x/tech/apis/_user-response-body.adoc[]
 
 This API is used to delete a Admin or Moderator. When a user is deleted they are marked as disabled.
 
-Once a user has been deleted, they may not be enabled through the use of an API. A deleted user can be enabled by a CleanSpeak admin user in the CleanSpeak Management Interface.
+Once a user has been deleted, they may not be enabled through the use of an API. A deleted user can be enabled by a CleanSpeak Admin user in the CleanSpeak Management Interface.
```
Comment (robotdan) [inline docs/3.x/tech/apis/admins-moderators.adoc:96]: Don't think we need to capitalize `Admin` unless it is a proper name.

File: docs/3.x/tech/apis/authentication.adoc [hunk 8]
```diff
@@ -6,20 +6,19 @@ description: Learn how the CleanSpeak APIs authenticate requests
...
 == Authentication
 
-CleanSpeak controls access to the API through the use of an API key. By default all APIs will return an `401 Unauthorized` response.
+CleanSpeak controls access to the API through the use of an API Key. By default all APIs will return an 401 Unauthorized response.
 
-To enable access to the API create one or more API keys in the CleanSpeak management interface. The API key is then supplied in the HTTP request using the Authorization header.  See API Keys for more information on adding additional keys.
+To enable access to the API create one or more API Keys in the CleanSpeak management interface. The API Key is then supplied in the HTTP request using the Authorization header.
 
-The following example demonstrates the HTTP Authorization header with an API key of `2524a832-c1c6-4894-9125-41a9ea84e013`.
+The following example demonstrates the HTTP Authorization Header with an API Key of `2524a832-c1c6-4894-9125-41a9ea84e013`.
```
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:13]: You have mixed usage of `Header` and `header`, I don't think it should be capitlized.

File: docs/3.x/tech/apis/authentication.adoc [hunk 9]
```diff
@@ -6,20 +6,19 @@ description: Learn how the CleanSpeak APIs authenticate requests
 
 == Authentication
 
-CleanSpeak controls access to the API through the use of an API key. By default all APIs will return an `401 Unauthorized` response.
+CleanSpeak controls access to the API through the use of an API Key. By default all APIs will return an 401 Unauthorized response.
 
-To enable access to the API create one or more API keys in the CleanSpeak management interface. The API key is then supplied in the HTTP request using the Authorization header.  See API Keys for more information on adding additional keys.
+To enable access to the API create one or more API Keys in the CleanSpeak management interface. The API Key is then supplied in the HTTP request using the Authorization header.
```
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:11]: You dropped this last sentence, was that on purpose?

File: docs/3.x/tech/apis/authentication.adoc [hunk 10]
```diff
@@ -6,20 +6,19 @@ description: Learn how the CleanSpeak APIs authenticate requests
 
 == Authentication
 
-CleanSpeak controls access to the API through the use of an API key. By default all APIs will return an `401 Unauthorized` response.
+CleanSpeak controls access to the API through the use of an API Key. By default all APIs will return an 401 Unauthorized response.
```
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:9]: You can add a back tick on `401` itself.

File: docs/3.x/tech/apis/authentication.adoc (hunk 9 above)
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:11]: Some places you are using back ticks for `Authorization`, we should be consistent.

File: docs/3.x/tech/apis/blacklist.adoc [hunk 11]
```diff
@@ -231,7 +229,7 @@ include::docs/3.x/tech/apis/_standard-delete-response-codes.adoc[]
 
 == Bulk Delete Dictionary Entries by Locale
 
-WARNING: This api is potentially catastrophically destructive and cannot be undone. Use care with this API.
+WARNING: This `API` is potentially catastrophically destructive and cannot be undone. Use this `API` with care.
```
Comment (robotdan) [inline docs/3.x/tech/apis/blacklist.adoc:232]: Why are you using back ticks here?

File: docs/3.x/tech/apis/authentication.adoc (hunk 9 above)
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:11]: Most places we are capitalizing `CleanSpeak management interface` you should be consistent.

File: docs/3.x/tech/apis/authentication.adoc [hunk 12]
```diff
@@ -6,20 +6,19 @@ description: Learn how the CleanSpeak APIs authenticate requests
...
+The following example demonstrates the HTTP Authorization Header with an API Key of `2524a832-c1c6-4894-9125-41a9ea84e013`.
 
 [source,properties]
 ----
 Authorization: 2524a832-c1c6-4894-9125-41a9ea84e013
 ----
 
-The following is a cURL example using the Authorization header with a base CleanSpeak URL of `https://example-cleanspeak-api.inversoft.io`.
-
+The following is a cURL example using the Authorization Header.
```
Comment (robotdan) [inline docs/3.x/tech/apis/authentication.adoc:20]: Header should not be capitalized.

File: docs/3.x/tech/apis/content.adoc [hunk 13]
```diff
@@ -81,7 +81,7 @@ include::docs/3.x/tech/apis/_moderate-content-response-body.adoc[]
 
 This API is used to send updates to existing content to CleanSpeak to be moderated. Only persistent content can be updated in CleanSpeak.
 
-Like the Filter Content API the same filtering is performed on content sent to this API. The difference is that instead of returning all matches found in the content, the Application filter rules are first applied to determine what action should be taken. Using this API allows you to modify the filter behavior without changing the way you send content to the filter. This allows for a much more powerful integration.
+Like the Filter Content API the same filtering is performed on content sent to this API. The difference is that instead of returning all matches found in the content, the Application Filter rules are first applied to determine what action should be taken. Using this API allows you to modify the filter behavior without changing the way you send content to the filter. This allows for a much more powerful integration.
```
Comment (robotdan) [inline docs/3.x/tech/apis/content.adoc:84]: Why is `Filter` uppercased?  You aren't doing this other places, and I don't think it should be capitalized.

File: docs/3.x/tech/apis/index.adoc [hunk 14]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
 title: API Overview
-description: An overview of the CleanSpeak APIs
+description: An overview of the CleanSpeak `APIs`
```
Comment (robotdan) [inline docs/3.x/tech/apis/index.adoc:4]: Why is there back ticks here?

File: docs/3.x/tech/apis/kids-chat.adoc [hunk 15]
```diff
@@ -8,49 +8,46 @@ description: Whitelist APIs that allow your application to access the whitelist
 
 [NOTE.info]
 --
-The whitelist is known as the kids chat on the frontend. In this document it will still be referred to as the whitelist.
+The Whitelist is known as the kids chat on the frontend. In this document it will still be referred to as the Whitelist.
```
Comment (robotdan) [inline docs/3.x/tech/apis/kids-chat.adoc:11]: `kids chat` is referred to as `Kids Chat` in the front end -or more specifically the CleanSpeak Management Interface.

File: docs/3.x/tech/apis/kids-chat.adoc [hunk 16]
```diff
@@ -8,49 +8,46 @@ description: Whitelist APIs that allow your application to access the whitelist
...
 --
 
-These APIs allow you to query and modify the whitelist filter state.
+These APIs allow you to query and modify the Whitelist Filter state.
```
Comment (robotdan) [inline docs/3.x/tech/apis/kids-chat.adoc:14]: Don't think we should upper case `Filter`

File: docs/3.x/tech/apis/kids-chat.adoc [hunk 17]
```diff
@@ -8,49 +8,46 @@ description: Whitelist APIs that allow your application to access the whitelist
...
 
-The whitelist is best understood as a deny first list of words people are allowed to use. Only when a user has matched
-an allowed word or phrase will the content be permitted. The whitelist also has its own set of blacklist rules called
-disallowed phrases that prevent combinations of whitelisted words being used together in ways that could be just as bad
-as using a blacklisted word.
+The Whitelist is best understood as a deny first list of words people are allowed to use. Only when a user has matched an allowed word or phrase will the content be permitted. The `Whitelist` also has its own set of `Blacklist` rules called disallowed phrases that prevent combinations of `Whitelisted` words being used together in ways that could be just as bad as using a blacklisted word.
```
Comment (robotdan) [inline docs/3.x/tech/apis/kids-chat.adoc:16]: Remove back ticks.

File: docs/3.x/tech/apis/kids-chat.adoc [hunk 18]
```diff
@@ -8,49 +8,46 @@ description: Whitelist APIs that allow your application to access the whitelist
...
+** <<Update an Allowed Word, Update an Allowed Word>>
 * Tags
-** <<Retrieve a Tag>>
-** <<Retrieve all Tags>>
-** <<Delete a Tag>>
-** <<Create a Tag>>
+** <<Retrieve a Tag,>>
+** <<Retrieve all Tags,>>
+** <<Delete a Tag,>>
+** <<Create a Tag,>>
```
Comment (robotdan) [inline docs/3.x/tech/apis/kids-chat.adoc:30]: You broke these, make sure you verify this renders before checking in.

File: docs/3.x/tech/apis/status.adoc [hunk 19]
```diff
@@ -17,7 +17,7 @@ This API provides status information about CleanSpeak with the help of the https
 --
 
 === Response
-The JSON response from the Status API is complex and subject to change. The specific contents of the JSON body are not documented here. If you choose to use this API for monitoring purposes you should primarily use the response code to indicate server health. If you receive a `200` you may consider CleanSpeak in a healthy state. The response body is intended for use by Inversoft support.
+The JSON response from the Status API is complex and subject to change. The specific contents of the JSON body are not documented here. If you choose to use this API for monitoring purposes you should primarily use the response code to indicate server health. If you receive a 200 you may consider CleanSpeak in a healthy state. The response body is intended for use by CleanSpeak support.
```
Comment (robotdan) [inline docs/3.x/tech/apis/status.adoc:20]: You removed back ticks on `200` - was that on purpose?

File: docs/3.x/tech/apis/status.adoc [hunk 20]
```diff
@@ -28,26 +28,26 @@ The JSON response from the Status API is complex and subject to change. The spec
...
 |500
-|The CleanSpeak server is not functioning properly. This could indicate that the database connectivity failed or one or more services within CleanSpeak failed. Consult the CleanSpeak link:../troubleshooting/[Troubleshooting] to learn more about the failure or contact Inversoft support for assistance.
+|The CleanSpeak server is not functioning properly. This could indicate that the database connectivity failed or one or more services within CleanSpeak failed. Consult the CleanSpeak link:../troubleshooting/[Troubleshooting] to learn more about the failure or contact CleanSpeak support for assistance.
 |===
 
 
 == Prometheus
 
-This API provides metrics similar to the Status Endpoint but responds in a format designed for link:https://prometheus.io[Prometheus].
+This AP provides metrics similar to the Status Endpoint but responds in a format designed for link:https://prometheus.io[Prometheus].
```
Comment (robotdan) [inline docs/3.x/tech/apis/status.adoc:37]: Typo

File: docs/3.x/tech/apis/system.adoc [hunk 21]
```diff
@@ -56,15 +56,15 @@ The following configuration is not contained in the backup and as such will also
 In the following cURL example a backup is requested and saved to a file named `backup.zip`. Note this example has omitted the required `Authorization` header, see link:authentication[Authentication].
 [source]
 ----
-curl -XGET 'https://example-cleanspeak-api.inversoft.io/system/backup' -o backup.zip
+curl -XGET https://example-cleanspeak-api.io/system/backup -o backup.zip
```
Comment (robotdan) [inline docs/3.x/tech/apis/system.adoc:59]: You broke this URL, `.io` is a TLD but not one registered to CleanSpeak.

File: docs/3.x/tech/apis/system.adoc [hunk 22]
```diff
@@ -56,15 +56,15 @@ The following configuration is not contained in the backup and as such will also
...
 ----
 --
 
 === Response
 
 [NOTE]
 ====
-Note that a successful response will not contain JSON but instead the response will have a `Content-Type` of `application/octet-stream`.
+Note that a successful response will not contain JSON but instead the response will have a Content-Type of `application/octet-stream`.
```
Comment (robotdan) [inline docs/3.x/tech/apis/system.adoc:67]: You removed the back ticks from `Content-Type` these should remain.

File: docs/3.x/tech/apis/system.adoc [hunk 23]
```diff
@@ -100,7 +100,7 @@ It is recommended that if you intend to use the backup and restore APIs in a pro
 The following cURL example assumes a file named `backup.zip` exists in the current directory. Note this example has omitted the required `Authorization` header, see link:authentication[Authentication].
 [source]
 ----
-curl -XPOST -H 'Content-Type: application/octet-stream' 'https://example-cleanspeak-api.inversoft.io/system/restore' --data-binary @backup.zip
+curl -XPOST -H 'Content-Type: application/octet-stream' 'https://example-cleanspeak-api.io/system/restore' --data-binary @backup.zip
```
Comment (robotdan) [inline docs/3.x/tech/apis/system.adoc:103]: Broken URL

File: docs/3.x/tech/apis/url.adoc [hunk 24]
```diff
@@ -15,39 +15,39 @@ Available since `3.25.0`
...
+The following APIs only operate on the active filter state. There is no approval process for the APIs and everything will be immediately added to the active filter state. Use the Management Interface to manage pending modifications.
 ====
 
 === Top Level Domains
 
-The following APIs are provided to manage Top Level Domains inside of CleanSpeak. A Top Level Domain or TLD is a unique name used by various agencies as the root domain of any website. An example of this is the `COM` TLD. CleanSpeak uses TLDs to identify URLs and this API exists so that you can modify which TLDs CleanSpeak identifies as URLs and their qualities so that bad urls can be easily tuned.
+The following APIs are provided to manage Top Level Domains inside of CleanSpeak. A Top Level Domain or TLD is a unique name used by various agencies as the root domain of any website. An example of this is the COM TLD. CleanSpeak uses TLDs to identify URLs and this API exists so that you can modify which TLDs CleanSpeak identifies as URLs and their qualities so that bad URLs can be easily tuned.
```
Comment (robotdan) [inline docs/3.x/tech/apis/url.adoc:23]: You removed back ticks on `COM`

**Review by robotdan — COMMENTED** [review]
I added a bunch of comments, but you're still missing a bunch of stuff. Please take another pass.

File: docs/3.x/tech/apis/authentication.adoc (hunk 9 above)
Comment (TharonWeighill) [reply docs/3.x/tech/apis/authentication.adoc:11]: Yes, It states to see API Keys for more information on adding additional keys, but that documentation doesn't exist. Or am I misunderstanding something?

**Pull request merged**